diff -urNp -x '*.orig' Pillow-6.2.2.org/setup.py Pillow-6.2.2/setup.py
--- Pillow-6.2.2.org/setup.py	2020-01-02 09:33:29.000000000 +0100
+++ Pillow-6.2.2/setup.py	2021-02-28 22:04:53.654351182 +0100
@@ -424,6 +424,8 @@ class pil_build_ext(build_ext):
 
         #
         # add platform directories
+        _add_directory(library_dirs, "/libx32")
+        _add_directory(library_dirs, "/usr/libx32")
 
         if self.disable_platform_guessing:
             pass
